client/src/index.jsx

```diff
@@ -56,8 +58,8 @@ class App extends React.Component {
 		})
 		.catch((err) => {
 			if (err) {
-				console.log('error in componentDidMount');
-				console.log(err);
+				console.error('error in componentDidMount');
+				console.error(err);
```

Comment:
Good to know console.error is a thing - similar to logger.error. Are there options, or just the two?

Reply:
https://developer.mozilla.org/en-US/docs/Web/API/console